airflow-ctl/src/airflowctl/api/client.py

```diff
@@ -263,7 +263,9 @@ def get_client(kind: ClientKind = ClientKind.CLI):
     """
     api_client = None
     try:
-        credentials = Credentials().load()
+        credentials = Credentials()
+        if kind == ClientKind.CLI:
```

Comment:
Maybe we can put this in credentials where we can check and act accordingly since kind is already defined in that file, and logic is there. What do you think?

Reply:
That's fair. Let's not update the load functionality then :)